scalameta/contrib/shared/src/main/scala/scala/meta/contrib/ScaladocParser.scala

```diff
@@ -29,8 +29,7 @@ object ScaladocParser {
           if (remainingScaladoc.trim.nonEmpty || remainingScaladoc.contains("\n\n")) {
             // Adds the parsed token to the list of tokens and parse the rest of the string recursively.
             if (remainingScaladoc.take(2) == "\n\n") {
-              List(p.value, DocToken(Paragraph)) ++ parseRec(
-                remainingScaladoc.dropWhile(_ == '\n'))
+              List(p.value, DocToken(Paragraph)) ++ parseRec(remainingScaladoc.dropWhile(_ == '\n'))
```

Comment:
Neat! So I'm longer gonna need this: https://github.com/xeno-by/scalamacros/blob/master/.scalafmt.conf#L6.